### ISSUE TITLE
Publish these images to docker.io and quay.io

### ISSUE THREAD

**Issue by lance** [boson-project/buildpacks#2]

**Comment by matejvasek** [comment]
@lance isn't quay.io enough?

**Comment by lance** [comment]
@matejvasek that's probably fine

**Comment by matejvasek** [comment]
@lance I pushed images to quay.io/boson, I pushed all produced images just to be sure, although it should have been sufficient to push builder and runner. I noticed one thing builder are distinguished by image name, but some other images are distinguished by tag (e.g. img-quarkus vs. img:quarkus). Is it ok? Also how we will version images?

**Comment by lance** [comment]
@matejvasek good questions. I modeled this off of the ["official" samples](https://github.com/buildpacks/samples/tree/master/stacks) and thought the naming was odd when I created them. I'm happy to consider a different approach if this way is confusing or doesn't make sense. It might be good to do some research on what other orgs (e.g. Heroku) are doing with their buildpacks.

Closing this issue for now since these images have been pushed. I've opened a new issue to figure out what we should do here. https://github.com/boson-project/buildpacks/issues/5
